src/translations/translations_EN.json

```diff
@@ -126,6 +126,12 @@
     },
     "edit_resource": "Edit resource",
     "files_and_license": {
+      "additional_files": {
+        "additional_files": "Additional files",
+        "additional_files_description": "In case you need more files in order to make the resource complete. For instance if you want both word and pdf, or multiple language files, Latex, formulas etc.",
```

Comment:
```suggestion
        "additional_files_description": "In case you need more files in order to make the resource complete. For instance if you want both a Word and a PDF document, or multiple language files, Latex files, formulas etc.",
```